docs/EMERGENCY.md

```diff
@@ -0,0 +1,17 @@
+# Emergency mode
+
+ES-Operator will not operate a non-green cluster. Therefore, there are certain situations that require human intervention to recover an Elasticsearch cluster running with ES-Operator.
+
+## Stopping the bleeding
+
+The first and foremost goal should be to get your cluster back into green state. The [cluster reroute API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-reroute.html#cluster-reroute-api-request-body) can be used to temporarily allocate an empty primary, until you have recovered your data.
+
+If, for whatever reason, this is not feasible, you will need to switch from auto-pilot to manual steering. For this, please do the following:
+
+## Manual Recovery Steps
+
+1. Stop the ES-Operator.
```

Comment:
I don't think most people will understand what to do for stopping an operator. It's better to explicitly mention that the es-operator deployment needs to be scaled down using the `kubectl scale ...` command.